GeeksCoreLibrary/Modules/PostalServices/NeDistri/Controllers/NeDistriController.cs

```diff
@@ -29,10 +30,35 @@ public async Task<IActionResult> GenerateShippingLabelAsync([FromQuery] Shipping
             return BadRequest("No order id specified");
         }
 
+        var types = model.LabelType.Split(',');
+        var coliAmounts = model.ColliAmount.Split(',');
+
+        if (types.Length != coliAmounts.Length)
+        {
+            return BadRequest("Amount of types and coliNumbers are not equal");
+        }
+
+        var labelRules = new List<LabelRule>();
+        
+        for (var i = 0; i < types.Length; i++)
+        {
+            var parseSuccess = Int32.TryParse(coliAmounts[i], out var coliAmount);
+
+            if (!parseSuccess)
```

Comment:
Please move the `TryParse` directly in the if-statement, since the resulted boolean is not used anywhere else.